lib/test.js

```diff
@@ -28,21 +29,16 @@ function Test(title, fn) {
       'string'
     );
   }
+  super(title, fn);
   this.type = 'test';
```

Comment:
Let's err on the side of caution ;)

```suggestion
  this.type = 'test';
  super(title, fn);
```

Reply:
This one is a similar issue as to the `super()` call from earlier, it might technically be the same issue in the grand scheme of things. Accessing `this` before calling `super()` is a `ReferenceError`.

<img width="1007" height="204" alt="Accessing This Before Super Demo" src="https://github.com/user-attachments/assets/ce14e5a1-a787-4b5f-8d02-e1e31c90eb7f" />